packages/catalog-search/src/SearchSuggestions.jsx

```diff
@@ -52,6 +56,28 @@ const SearchSuggestions = ({
             ))
         }
       </div>
+      )}
+      {/* Currently (Feb 2023) it is not possible to redirect to the learner portal for exec ed content so only display
+       if the redirect is disabled */}
```

Comment:
[curious] Is coming back to update this redirect logic once Learner portal can display exec ed content already ticketed somewhere?

Reply:
it is not, but I can do it

Reply:
Thanks! Re-posting here: https://2u-internal.atlassian.net/browse/ENT-7171